crates/parry3d-f64/Cargo.toml

```diff
@@ -59,6 +60,11 @@ cust_core = { version = "0.1", optional = true }
 spade = { version = "2", optional = true } # Make this optional?
 rayon = { version = "1", optional = true }
 bytemuck = { version = "1", features = ["derive"], optional = true }
+rstar = "0.12.0"
+obj = { version = "0.10.2", optional = true }
+
+[target.'cfg(not(target_os = "cuda"))'.dependencies]
+cust = { version = "0.3", optional = true }
```

Comment:
This should be removed. This is probably an artifact introduced by the merge with `main`.
```suggestion
```